src/pages/Autolending/AutolendingWhen.vue

```diff
@@ -36,39 +36,44 @@
 				>
 					<div class="when-inputs-wrapper">
 						<lend-timing-dropdown />
-						<kv-radio
-							data-test="is-autolending-donation-on"
-							id="is-autolending-donation-on"
-							radio-value="on"
-							v-model="donationToggle"
-						>
-							Include a donation to Kiva of
-							<kv-dropdown-rounded v-model="donation">
-								<option :value="0">
-									0%
-								</option>
-								<option :value="5">
-									5%
-								</option>
-								<option :value="10">
-									10%
-								</option>
-								<option :value="15">
-									15%
-								</option>
-								<option :value="20">
-									20%
-								</option>
-							</kv-dropdown-rounded>
-						</kv-radio>
-						<kv-radio
-							data-test="is-autolending-donation-off"
-							id="is-autolending-donation-off"
-							radio-value="off"
-							v-model="donationToggle"
-						>
-							No donation to Kiva
-						</kv-radio>
+						<p v-if="hasAutoDepositDonation && donation === 0">
```

Comment:
@remoun we should remove this `donation === 0` as new users who have AutoDepositDonation will not have a donation of 0

Reply:
It seems better to tweak our initialization code to not set auto-lending donations when there's an auto-deposit donation -- in that indeterminate state, incorrectly hiding the edit UI seems worse; plus when users actually attempt to set the donationPercentage it would get reset to 0. I believe I've handled all the call sites for https://kiva.atlassian.net/browse/AUTO-44.

Reply:
Hiding the additional donation Ui if they have a donation set elsewhere is the correct path. It's highly likely that the new lender would have > 0 default percentage set so they will see a Ui that they are ineligible with the code above.

I think you could potentially solve this in the initialization phase by running an edit mutation in the mounted hook to zero out the donation in client state. If the visitor chooses to complete set up and "save" their auto-lend profile, their donation would then be stored as 0. This would allow the above code to work for both scenarios.

Reply:
Note that [Monthly Good setup explicitly sets `donationPercentage=0`](https://github.com/kiva/kiva/blob/c33c23a493ab1921c0b18184c0256a5fca975d50/libs/kv/lib/monthly_good.php#L74) and [so does `setAutolendingEnabled`](https://github.com/kiva/kiva/blob/c33c23a493ab1921c0b18184c0256a5fca975d50/httpdocs/classes/API/FundAccount.php#L559)

I started adding the mutation, but realized that that would just update the client-side state, and create a consistently misleading UI. So I'm inclined to keep it with the `hasAutoDepositDonation && donation === 0` to handle the default case and degrade gracefully for other combinations.

Reply:
Added a comment summarizing this rationale.

Reply:
Ok I see more clearly where you are coming from now. Any case where this still occurs should be handled with additional messaging and state handling in a new ticket. We'll likely want to create a new ticket for how to handle lenders that arrive to this page who have autolending off but an amount already set on their autolend_profile too.

Reply:
I'm pretty sure we're good in that case too. We reset the AL donation when we turn on auto-lending if they have an auto-deposit donation, as [that flow goes `setAutolendingEnabled`](https://github.com/kiva/kiva/blob/c33c23a493ab1921c0b18184c0256a5fca975d50/httpdocs/classes/API/FundAccount.php#L559).

While trying this out in my VM as an MG subscriber, `/settings/autolending` redirected to `/settings/credit`, and the latter told me auto-lending is not available for MG subscribers. So I think that covers all the paths.